### DISCUSSION TITLE
Social login for rest endpoints users

### DISCUSSION THREAD

**Discussion by EsmaeelNabilM** [directus/directus#10384] [Feature Requests]
Since `Facebook` and `google` has `depreciated` embedded web-views auth, and we can’t use the same default social login configuration generated url’s.

`How` should we integrate with Facebook and Google Mobile Sdk’s to create directus users as social accounts/users ? 

What is the recommended flow and steps should i use if i needed to create a social/user from a mobile app?  

https://developers.facebook.com/blog/post/2021/06/28/deprecating-support-fb-login-authentication-android-embedded-browsers/

**Comment by rijkvanzanten** [comment]
I'm not sure if I understand the question.. Are you asking how to generate a user in Directus when somebody logs in through the social provider?

**Reply by EsmaeelNabilM** [reply]
Exactly but from a mobile app

**Reply by rijkvanzanten** [reply]
If you use the OAuth2 / OpenID / LDAP login flows, it's already possible to have Directus automatically create the user when they login the first time: https://docs.directus.io/configuration/config-options/#oauth-2-0

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
